sova/cli/commands/maintain.py

```
@@ -0,0 +1,129 @@
+    if not config.github_repo:
+        console.print("[red]No github_repo configured in sova.toml[/red]")
+        raise typer.Exit(code=1)
+
+    if dry_run:
+        console.print(f"[bold]Inspecting Dependabot PRs for {config.github_repo}...[/bold]")
+
+        from sova.git.pr import list_open_prs
+
+        try:
+            all_prs = await list_open_prs(repo=config.github_repo, github_user=config.github_user)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)